### ISSUE TITLE
separator in input csv file for generate_bids_dataset

### ISSUE THREAD

**Issue by SylvainTakerkart** [INT-NIT/BEP032tools#138]
Quick issue before I abandon my keyboard for the day... From what I've just seen, if the separator is a tab in the csv file (instead of a comma), it yields an error...

This first needs to be rechecked (for instance by modifying the example.csv file to replace the commas by tabs, and running the generator on it)!

And if it gets confirmed, we will have to fix this!

**Comment by JuliaSprenger** [comment]
Yes, I guess I assumed a consistency between the separator indicated by the extension and the one used in the file. So if you also want to be tabs a valid separator then we should add support for `tsv`.
